http.go

```diff
@@ -40,9 +41,13 @@ type ServerHandler struct {
 	modifiedWRPCount         metrics.Counter
 	incomingQueueDepth       int64
 	maxOutstanding           int64
+	incomingQueueLatency     metrics.Histogram
+	now                      func() time.Time
 }
 
 func (sh *ServerHandler) ServeHTTP(response http.ResponseWriter, request *http.Request) {
+	startTime := sh.now()
+	eventType := "Unknown"
```

Comment:
I would suggest making this an unexported const at the top of `metrics.go`.  Then the tests can use it too.

```go
     unknownEventType = "unknown"
```